modules/social_features/social_featured_items/config/update/social_featured_items_update_10200.yml

```diff
@@ -0,0 +1,30 @@
+field.field.paragraph.featured_item.field_featured_item_description:
```

Comment:
Since we add this to 10.3 milestone, need to rename the number to start with 103

Reply:
fixed